test/kubernetes_unittest.cc

```diff
@@ -185,7 +191,7 @@ TEST_F(KubernetesTest, GetPodMetadata) {
   Configuration config(std::stringstream(
     "KubernetesClusterName: TestClusterName\n"
     "KubernetesClusterLocation: TestClusterLocation\n"
-    "MetadataApiResourceTypePerarator: \",\"\n"
+    "MetadataApiResourceTypeSeperator: \",\"\n"
```

Comment:
Make that `MetadataApiResourceTypeSeparator` (here and below).
Sigh, the config revamp in #127 would allow catching invalid config options.

Reply:
This still needs to be fixed.

Reply:
Fixed, sorry about that